draft-ietf-core-cf-reg-update.md

```diff
@@ -82,7 +82,10 @@ This document uses the terms "media type", "content coding", "content-type", and
 
 # Examples for Erroneous Registrations
 
-This section contains examples of registration requests for a CoAP Content-Format with identifier 64999 in the FCFS range of the "CoAP Content-Formats" registry, as defined in {{Section 12.3 of -coap}} and revised according to {{Err4954}}, which must not be allowed to succeed.
+This section provides examples of registration requests for the "CoAP Content-Formats" Registry (as defined in {{Section 12.3 of -coap}} and revised according to {{Err4954}}) that must be denied.
```

Comment:
maybe 'must'->'would' ?  But fine either way.

Reply:
"would be denied" is what would not happen with the current (existing) rules; it would be accepted.
"must be denied" is what we would like to happen with the fixed procedure.

Maybe it's more clear if we write "that need to be denied, but would be approved by the present procedure" ?

Or the short version "that would need to be denied".  3 verbs in 6 words is maybe stretching the limit :)

Reply:
My two cents:

> _”[...] that are invalid but would be approved under the current procedure.”_

Reply:
@thomas-fossati That sounds good - I'll let you pick what to add!